python/sglang/srt/layers/quantization/qqq.py

```diff
@@ -0,0 +1,300 @@
+# Adapted from https://github.com/vllm-project/vllm/blob/v0.6.3.post1/vllm/model_executor/layers/quantization/qqq.py
+
+import logging
+from typing import Any, Dict, List, Optional
+
+import torch
+from torch.nn.parameter import Parameter
+from torchao.ops import marlin_qqq_gemm
```

Comment:
```
ImportError: cannot import name 'marlin_qqq_gemm' from 'torchao.ops'
```
It should be due to the version, the current release of torchao (v0.6.1) does not include qqq.

Reply:
Perhaps we can introduce qqq in the next PR after torchao releases a new version, how about that @HandH1998

Reply:
ok